Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -111,6 +116,15 @@ public double CompactPaneLength
         /// <summary>
         /// Gets or sets the Brush to apply to the background of the Pane area of the control.
         /// </summary>
+        public Brush AccentColor
```

Comment:
I would rather have this property called "HighlightColor" rather than AccentColor. AccentColor gives no clear indication what it's actually used for.